zokrates_book/src/reference/Testing.md

```diff
@@ -1,6 +1,17 @@
 # Testing
 
+Before running any tests, make sure make sure your `ZOKRATES_HOME` environment variable is set correctly.
+It has to point to `zokrates_stdlib/stdlib/`
+
 ## Unit tests
+In ZoKrates, unit tests comprise
+- internal tests for all zokrates crates
+- compilation tests for all examples in the `$ZOKRATES_HOME/` zokrates_cli example folder. These tests only ensure that the examples compile correctly.
```

Comment:
You're talking about a different `ZOKRATES_HOME` here (root of the project).
```suggestion
- compilation tests for all examples in the `zokrates_cli/examples`. These tests only ensure that the examples compile.
```

Reply:
Great, good catch!